File: erpnext/accounts/report/purchase_invoice_trends/purchase_invoice_trends.py
# Copyright (c) 2015, NETMANTHAN TECHNOLOGIES. and Contributors
# License: GNU General Public License v3. See license.txt


from erpnext.controllers.trends import get_columns, get_data


def execute(filters=None):
	if not filters:
		filters = {}
	data = []
	conditions = get_columns(filters, "Purchase Invoice")
	data = get_data(filters, conditions)

	return conditions["columns"], data

File: erpnext/accounts/report/purchase_invoice_trends/purchase_invoice_trends.js
// Copyright (c) 2015, Sparrownova Technologies and Contributors
// License: GNU General Public License v3. See license.txt

frappe.require("assets/erpnext/js/purchase_trends_filters.js", function() {
	frappe.query_reports["Purchase Invoice Trends"] = {
		filters: erpnext.get_purchase_trends_filters()
	}
});
